rbac/providers/azure/delta_inbound_sync.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
```

Comment:
Why does this file get special UTF-8 treatment?

Reply:
sorry, force of habit whenever I write python files. I'll keep them out